tests/sample_params.aws.bam.yml

```diff
@@ -0,0 +1,6 @@
+sample_id: NA12878
+cram: s3://sg100k-benchmarking-data/datasets/HG001/1000genomes-dragen/NA12878.bam
```

Comment:
again GRIDS resources hardcoded in a public repo

Reply:
indeed @justinjj24 can you fix this before I acknowledge this PR. @mhebrard can you help @justinjj24 if it isnt obvious how to fix this ?

Reply:
fixed to cram: s3://1000genomes-dragen-3.7.6/data/individuals/hg38-graph-based/NA12878/NA12878.bam